.rubocop.yml

```diff
@@ -31,3 +31,9 @@ Lint/BlockAlignment:
 
 Lint/InheritException:
   Enabled: false
+
+Style/GuardClause:
+  Enabled: false
```

Comment:
I remember consulting with the team and we decided to be more flexible about when to use them. If you see a place in the code where you think I should be using them, please raise it up!